apps/system/style/utility_tray/utility_tray.css

```diff
@@ -132,6 +132,15 @@
   background: hsla(0, 0%, 20%, 1) url(images/grippy.png) no-repeat center / 7.2rem;
 }
 
+#screen.software-button-enabled.utility-tray #utility-tray > #utility-tray-grippy:after {
```

Comment:
can we maybe just toggle a pointer-event when the `screen.utility-tray` is enabled/disabled?
(to prevent extra layout work when we start dragging the utility tray)